tunedev/Banka-react
# #167966111 make account summary page functional

#### What does this PR do?
Adds functionality to the summary page

#### Description of Task to be completed?
> - add functionality to the user account summary page

#### How should this be manually tested?
> - Clone this branch
> - Then run `npm install` and `npm test'
> - Then run `npm run dev' 
> - Click on the Sign-up button on the Landing page
> - Fill the form with unique details and hit submit to sign up
> - you should be taking to a page where all your accounts summary is displayed

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#167966111](https://www.pivotaltracker.com/story/show/167966111)

#### Screenshots (if appropriate)
NA
#### Questions:
NA
